test/mandrill-transport.js

```diff
@@ -165,4 +163,71 @@ describe('MandrillTransport', function() {
       });
     });
   });
+
+  describe('#send attachments usage', function() {
+    var nodemailer = require('nodemailer'),
+      transport = mandrillTransport(),
+      client = transport.mandrillClient,
+      wrappedTransport = nodemailer.createTransport(transport),
+      sendOptions;
+
+    before(function() {
+      sinon.stub(client.messages, 'send', function(options, cb) {
+        sendOptions = options;
+        cb([]);
+      });
+    });
+
+    beforeEach(function() {
+      sendOptions = {};
+    });
+
+    after(function() {
+      client.messages.send.restore();
+    });
+
+    it('attachments object', function(done) {
+      wrappedTransport.sendMail({
```

Comment:
dont merge yet, i thought of something that needs to be tested:

i should call sendMail with full details (from, to etc...) and make sure that it doesn't somehow interfere with the attachments logic.